src/advanced/bitcoin-anchoring.md

```diff
@@ -168,24 +168,25 @@ anchoring transaction only if the previous anchoring chain failed
   anchoring transaction and the hash of the corresponding Exonum block. The
   first anchoring transaction in the chain uses the UTXO of the funding
   transaction
-- Every validator creates an Exonum transaction containing a new proposed
-  anchoring transaction and its vote for said transaction. The validators commit
-  these transactions to the Exonum blockchain
+- Every validator node has an external sync [util][btc_anchoring_sync] that
+  creates containing a new proposed anchoring transaction and its vote for
+  said transaction. The validators commit these transactions to
+  the Exonum blockchain
 - When such transaction is executed, its signature is stored in the
   corresponding anchoring service table. When the number of signatures for the
   same anchoring proposal reaches `+2/3` value, said anchoring transaction
   appears in the table of anchoring transactions
-- A handler performs synchronization between Exonum network and Bitcoin network
-  for availability of uncommitted anchoring transactions and sends all such
-  anchoring transactions to Bitcoin. Therefore, even if some committed anchoring
-  transactions are lost from the network due to a fork, the handler will send
-  them to Bitcoin once again.
+- A sync [util][btc_anchoring_sync] performs synchronization between Exonum
+  network and Bitcoin network for availability of uncommitted anchoring transactions
+  and sends all such anchoring transactions to Bitcoin. Therefore, even if some
+  committed anchoring transactions are lost from the network due to a fork,
+  the handler will send them to Bitcoin once again.
 
 ## Setup and Configuration
 
 Anchoring requires additional
 [global and local configuration parameters](../architecture/configuration.md) to
-be set.
+be set. [Here][newbie-guide] you can find setup and deploy guide for newbies.
```

Comment:
Why isn't this called simply a setup guide? As far as I see, there are no other setup guides 🙃

Reply:
It's because of me. When I wrote it, there was a primary (outdated though) setup guide which had much less details. So I called it a "guide for newbies", since it was detailed to the last command. But later it became the primary thing, but was not renamed...